### PR TITLE
deposit: fix JSONSchema dependency paths

### PR DESCRIPTION
**Pull request by drjova**
* FIX Adds the right paths to `uft8` and `base64` plugins.

Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

**Comment by jirikuncar** [comment]
@drjova can you please rebase. I have pushed one commit meantime. Thanks

**Comment by drjova** [comment]
done!

File: invenio_deposit/static/js/deposit/jsonwidget.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
  */
 
-require(['jquery', 'base64', 'utf8', 'select2'], function($, base64, utf8, _select2) {
+require(['jquery', 'vendors/base-64/base64', 'vendors/utf8/utf8', 'select2'], function($, base64, utf8, _select2) {
```
Comment (jirikuncar) [inline invenio_deposit/static/js/deposit/jsonwidget.js]: @drjova what about breaking the line?

File: invenio_deposit/static/js/deposit/jsonwidget.js [hunk 2]
```diff
@@ -17,7 +17,13 @@
  * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
  */
 
-require(['jquery', 'base64', 'utf8', 'select2'], function($, base64, utf8, _select2) {
+require([
+    'jquery',
+    'vendors/base-64/base64',
+    'vendors/utf8/utf8',
+    'select2'
+  ],
```
Comment (jirikuncar) [inline invenio_deposit/static/js/deposit/jsonwidget.js]: whitespace at the end of line

**Comment by drjova** [comment]
:+1:

**Pull request merged**